server/main.go

```diff
@@ -38,6 +38,10 @@ func getWebHandler(activeFileManager *ActiveFileManager, fileStore FileStore) ht
 		method := req.Method
 		path := urlPathToArray(req.URL.Path)
 
+		res.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
If this isn't necessary, let's not open up this can of worms. When it's time to add auth, it'll be possible for any site to force a user to make these requests while being authenticated as them. I'm only willing to accept this as something to be careful about later if there's a good reason for it. Currently, I don't think there is.

The difference between an app client doing this and a browser is that running an app client requires the user to authenticate to the app before it can make the request. With the browser, the user may be looking at pictures of cats, not knowing that their InstantShare session is being hijacked. It's not a big deal now, but I like to design for the future and for simplicity.